management-interface.go

```diff
@@ -558,6 +558,65 @@ func WifiNetworkHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "wifi-networks.html", wifiProps)
 }
 
+// Return info on the packages that are currently installed on the device.
+func getInstalledPackages() (string, error) {
+	var out []byte
+	err := error(nil)
+	if runtime.GOOS != "windows" {
```

Comment:
how about just returning immediately if GOOS is "windows"? It'll make the whole method clearer. For example, you wouldn't need to declare `err` (just use `:=` on the `Command` call)

Reply:
really this should just be `if runtime.GOOS != "linux" as dpkg is only going to work on Linux.